modules/cache-common/src/internalClusterTest/java/org/opensearch/cache/common/tier/TieredSpilloverCacheStatsIT.java

```diff
@@ -238,7 +237,7 @@ public void testInvalidLevelsAreIgnored() throws Exception {
         internalCluster().startNodes(
             1,
             Settings.builder()
-                .put(TieredSpilloverCacheIT.defaultSettings(HEAP_CACHE_SIZE_STRING))
+                .put(TieredSpilloverCacheIT.defaultSettings(HEAP_CACHE_SIZE_STRING, TieredSpilloverCacheIT.getNumberOfSegments()))
```

Comment:
`getNumberOfSegments` is only used by caching related classes (IT/UT), so putting inside a common ```OpenSearchIntegTestCase``` did not make much sense to me as it will not be used elsewhere. Let me know what you think.

Reply:
Actually makes sense to have a common base IT class for tiered cache. Will make change.

Reply:
Maybe we can duplicate the method across classes, since it is small anyway!